server/src/main/java/io/spine/server/model/DuplicateCommandHandlerError.java

```diff
@@ -51,22 +55,67 @@ private static String fmt(CommandHandlingClass duplicatingClass,
         StringBuilder builder = new StringBuilder(512);
-        builder.append(format("The class %s declares handler methods for commands that are " +
-                              "already handled by other classes.", duplicatingClass));
+
+        builder.append(format("The class `%s` declares handler ", duplicatingClass));
+
+        // Do we have more than one command to report?
+        long totalDuplicatedCommands =
+                registeredHandlers.keySet()
+                                  .stream()
+                                  .mapToLong(Collection::size)
+                                  .sum();
+        checkState(totalDuplicatedCommands >= 1);
+        builder.append(
+                totalDuplicatedCommands == 1
+                ? "method for the command which is "
+                : "methods for commands that are "
+        );
+        builder.append("already handled by ");
+
+        // How many already handling classes do we have?
+        long totalHandlingClasses = registeredHandlers.values()
+                                                      .size();
+        builder.append(
+                totalHandlingClasses == 1
+                ? "another class."
+                : "other classes."
+        );
+        // Period. New line.
+        String newLine = format("%n");
+        builder.append(newLine);
+
+        // Now list the commands and their handlers.
         for (Set<CommandClass> commandClasses : registeredHandlers.keySet()) {
+            builder.append(newLine);
             if (commandClasses.size() > 1) {
                 builder.append(" Commands ");
-                builder.append(Joiner.on(", ").join(commandClasses));
+                List<String> commandsBackTicked =
+                        commandClasses.stream()
+                                      .map(DuplicateCommandHandlerError::backTick)
+                                      .collect(toList());
+                builder.append(
+                        Joiner.on(", ")
+                              .join(commandsBackTicked)
+                );
                 builder.append(" are handled by ");
             } else {
                 // One command.
                 builder.append(" The command ");
-                builder.append(commandClasses.iterator().next());
+                CommandClass cmdClass = commandClasses.iterator()
+                                                      .next();
+                builder.append(backTick(cmdClass));
                 builder.append(" is handled by ");
             }
-            builder.append(registeredHandlers.get(commandClasses));
+            CommandHandlingClass handlingClass = registeredHandlers.get(commandClasses);
+            builder.append(backTick(handlingClass));
+            builder.append('.');
         }
         return builder.toString();
     }
+
+    private static String backTick(Object obj) {
```

Comment:
I have looked through a [couple](https://en.wiktionary.org/wiki/backtick) of [different](https://www.techopedia.com/definition/17836/backtick) [sources](https://en.wikipedia.org/wiki/Grave_accent), and it seems that there is a `backtick`, but there isn't a `back tick`. 

So this method should probably be `private static String backtick(Object obj)`.